src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -134,7 +134,7 @@ export const getTrackType = (trackId: string) => {
     return null;
   }
 
-  if (trackId.startsWith('gene')) {
+  if (trackId.startsWith('gene') || trackId.startsWith('focus')) {
```

Comment:
should it be `.startsWith` or `.match`?

Reply:
startsWith is good for the first part of the if; but the second part of the if should be just an equality comparison (`trackId === 'focus'`)

Reply:
sorry i meant the changes that was made (the second part) and yeah equality is better. Also whats the reason we are keeping the first condition `trackId.startsWith('gene')` since we have replaced all occurence of `gene-*`

Reply:
> Also whats the reason we are keeping the first condition trackId.startsWith('gene') since we have replaced all occurence of gene-*

We only have full control over the name of the focus track. The if-condition above is checking for both the focus track and all the other gene-and-transcripts tracks that are coming from the track api:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/6834224/176636625-8a134ddc-c7e5-401b-ba69-ec5c83f21df8.png">

We talked about changing the way of how we identify the type of a track, because track ids will stop being meaningful and human-readable in the foreseeable future. But that's for the future and will need proper coordination with the BE team.